server/dhcp.py

```diff
@@ -121,16 +121,17 @@ def __init__(self):
 
     def _delay_response_thread(self):
         while not self.closed:
-            p = self.queue.get()
             if self.closed:
                 break
-            t, func, args, kw = p
-            now = time.time()
-            if now < t:
-                time.sleep(0.01)
-                self.queue.put(p)
-            else:
-                func(*args, **kw)
+            if not self.queue.empty():
+                p = self.queue.get()
```

Comment:
Hello, great thanks for your great library.
Just have two points for this ISSUE( though it already closed)
1) what if using [EAFP!](https://docs.python.org/3/glossary.html#term-eafp) style and adding timeout=1 rather than using LBYL style and directly continue the while loop without any delaying or sleep? 
The updated code will look like this:
```
try:
                data,addr,recvTimestamp = taskQueue.get(timeout=1)
                recvPacket = NTPPacket()
                recvPacket.from_data(data)
                timeStamp_high,timeStamp_low = recvPacket.GetTxTimeStamp()
                sendPacket = NTPPacket(version=3,mode=4)
                sendPacket.stratum = 2
                sendPacket.poll = 10
                '''
                sendPacket.precision = 0xfa
                sendPacket.root_delay = 0x0bfa
                sendPacket.root_dispersion = 0x0aa7
                sendPacket.ref_id = 0x808a8c2c
                '''
                sendPacket.ref_timestamp = recvTimestamp-5
                sendPacket.SetOriginTimeStamp(timeStamp_high,timeStamp_low)
                sendPacket.recv_timestamp = recvTimestamp
                sendPacket.tx_timestamp = system_to_ntp_time(time.time())
                socket.sendto(sendPacket.to_data(),addr)
                print("Sended to %s:%d" % (addr[0],addr[1]))
            except queue.Empty:
                continue
```

2) Do you think new release should be issued? I download the latest RELEASE version seems do not contain this fix 

What do you think? Thanks in advance

Reply:
Hi @greggzj, (1) you can open a pull request with the changes you would like to see and we can discuss them there.
(2) I also think that this fix is not yet in a new release. However, it is in the master branch which you can download, too. Yep, I make a new release.

Reply:
Thanks, I will create a pull request later. Also thanks for the new release